subprojects/problems/src/main/java/org/gradle/api/problems/Problem.java

```diff
@@ -30,11 +30,13 @@
 @Incubating
 public interface Problem {
     /**
-     * Returns the problem type.
+     * Returns the problem category.
      *
-     * @return the problem type
+     * @return the problem category
+     *
+     * @since 8.5
      */
-    String getProblemType();
+    String getProblemCategory();
```

Comment:
❌ I still think we should return a `ProblemCategory` type here and not just a string representation.